R/helpers.R

```diff
@@ -1044,3 +1044,30 @@ h_find_interval <- function(..., replacement = -Inf) {
   x <- findInterval(...)
   ifelse(x == 0, yes = replacement, no = x)
 }
+
+
+
+#'unpack stopping rules and return list
+#'
+#'@description
+#'
+#'recursively unpack nested stopping rules logical value and label given
+#'
+#'@param stopit_tree object from simulate method
+
+h_unpack_stopit <- function(stopit_tree) {
```

Comment:
can you please add unit tests covering all if/else paths for this function?

Reply:
Robert optimized the function so there is only one if condition left now.